spec/spec_helper.rb

```diff
@@ -18,5 +18,16 @@
 require 'active_support/core_ext/string'
 require 'pry'
 require 'support'
+require "pathname"
 
 I18n.enforce_available_locales = true
+
+SPEC_DIR = Pathname.new(File.dirname(__FILE__))
+LOG_DIR = SPEC_DIR.join("../tmp/logs")
+
+RSpec.configure do |c|
+  c.before :each do
+    FileUtils.rm_rf LOG_DIR
+    FileUtils.mkdir_p LOG_DIR
```

Comment:
I don't like IO in the tests. Can we use in-memory logger to avoid this?

Reply:
Maybe [this](https://stackoverflow.com/questions/1484129/ruby-send-logger-messages-to-a-string-variable) would work.